### PR TITLE
Dependencies and Scala Version Update

### PR DESCRIPTION
**Pull request by kiroco12**
Made modifications in the following files:
- build.sbt
- project/plugins.sbt
- project/ProjectPlugin.scala
This modifications updates Scala version of the project and its dependencies.

**Note:** Project can be imported, but code files wasn't modified, so this won't compile. Next step are code modifications.

### CODE REVIEW

File: project/ProjectPlugin.scala [hunk 1]
```diff
@@ -59,11 +68,53 @@ object ProjectPlugin extends AutoPlugin {
...
+        %("slf4j-simple", V.slf4jSimple),
+        %%("http4s-dsl", V.http4s),
+        %%("http4s-blaze-server", V.http4s),
+        %%("http4s-blaze-client", V.http4s),
+        %%("http4s-circe", V.http4s),
+        %("config"),
+        %%("jwt-core", V.jwtCore),
+        %%("scalatest", V.scalatest) % "test"
+      ),
+      addSbtPlugin("io.get-coursier" % "sbt-coursier" % "2.0.0-RC3-2")
```
Comment (juanpedromoreno) [inline project/ProjectPlugin.scala]: Is that the latest version of coursier? In any case, I'd define a variable for it too

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=0.13.12
\ No newline at end of file
+sbt.version=1.2.7
```
Comment (juanpedromoreno) [inline project/build.properties]:
```suggestion
sbt.version=1.2.8
```

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,5 +1,7 @@
...
-addSbtPlugin("com.typesafe.sbt"  % "sbt-native-packager" % "1.2.0-M8")
-addSbtPlugin("com.eed3si9n"      % "sbt-assembly"        % "0.14.4")
-addSbtPlugin("se.marcuslonnberg" % "sbt-docker"          % "1.4.1")
-addSbtPlugin("com.47deg"         % "sbt-org-policies"    % "0.5.13")
\ No newline at end of file
+addSbtPlugin("com.typesafe.sbt"  % "sbt-native-packager" % "1.4.1")
+addSbtPlugin("com.eed3si9n"      % "sbt-assembly"        % "0.14.10")
+addSbtPlugin("com.eed3si9n"      % "sbt-buildinfo"       % "0.9.0")
+addSbtPlugin("se.marcuslonnberg" % "sbt-docker"          % "1.5.0")
+addSbtPlugin("com.47deg"         % "sbt-org-policies"    % "0.12.0-M1")
```
Comment (juanpedromoreno) [inline project/plugins.sbt]:
```suggestion
addSbtPlugin("com.47deg"         % "sbt-org-policies"    % "0.12.0-M2")
```

File: project/ProjectPlugin.scala [hunk 4]
```diff
@@ -84,20 +136,21 @@ object ProjectPlugin extends AutoPlugin {
...
       parallelExecution in Test := false,
       cancelable in Global := true,
-      headers := Map(
-        "scala" -> (HeaderPattern.cStyleBlockComment,
-        s"""|/*
+      headerLicense := Some(
+        HeaderLicense.Custom(
+          s"""|/*
             | * scala-exercises - ${name.value}
             | * Copyright (C) 2015-2016 47 Degrees, LLC. <http://www.47deg.com>
```
Comment (calvellido) [inline project/ProjectPlugin.scala]:
```suggestion
            | * Copyright (C) 2015-2019 47 Degrees, LLC. <http://www.47deg.com>
```

Very minor thing, but I think we could be updating the copyright now that we are into this.

**Pull request merged**